### ISSUE TITLE
Copy test URL to clipboard

### ISSUE THREAD

**Issue by zuphilip** [zotero/scaffold#80]
I would like to have still another feature after #77: A possibility to copy the url of the a selected test to the clipboard, which I then can also open in my usual browser to use the browser tools etc.

However, I have no idea how to do a copy to clipboard functionality in XUL. Is this easily possible? For another project (not XUL) I used the library [clipboard.js](https://clipboardjs.com/). Or are there native Javascript/XUL functions for that?

**Comment by dstillman** [comment]
`Zotero.Utilities.Internal.copyTextToClipboard(str)` will do it for you, and just having Ctrl/Cmd-C work while a URL is selected would be a pretty clean option, but if the goal is just to open it in a browser then we could use `Zotero.launchURL(url)`. The buttons are getting out of hand, but that (and other actions) could go in a menu with a bound key.

**Comment by zuphilip** [comment]
Thank you very much for the information! Is it okay to first come up with a new button and try the redesign in another issue/PR? I could imagine a context menu for most of the buttons in the Testing tab.

**Comment by dstillman** [comment]
Sure.

(And a context menu would certainly make sense for this one.)

**Comment by zuphilip** [comment]
Implemented in #83
